src/Events/QR/Shortcode.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * The Shortcode class for the QR module.
+ *
+ * @since TBD
+ */
+
+namespace TEC\Events\QR;
+
+use Tribe\Shortcode\Shortcode_Abstract;
+use TEC\Common\QR\QR;
+
+/**
+ * Class Shortcode
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\QR
+ */
+class Shortcode extends Shortcode_Abstract {
+	/**
+	 * The shortcode tag.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $slug = 'tec_event_qr';
+
+	/**
+	 * {@inheritDoc}
+	 *
+	 * @var array
+	 */
+	protected $default_arguments = [
+		'mode' => 'current',
+		'id'   => '',
+		'size' => 6,
+	];
+
+	/**
+	 * {@inheritDoc}
```

Comment:
It would be nice to copy the real docblock from the parent instead of using `@inheritDoc`

Reply:
Reminder that `{@inheritDoc}` only inherits the description.